### ISSUE TITLE
specific versions of tensorflow, python, keras, etc.

### ISSUE THREAD

**Issue by gepsuuu** [U-rara/Crack-Detection-Mask-RCNN#2]
Thank you for this very helpful repository. But may I ask for specific versions of tensorflow, python, and keras required to run the code since I am having version compatibility issues? Thanks
